addons/docs/blocks.js

```diff
@@ -1 +1 @@
-module.exports = require('./dist/blocks');
+export * from './dist/esm/blocks';
```

Comment:
Why this one isn't a `module.exports` anymore?

Reply:
hmm good question, don't think this is right

Reply:
@hipstersmoothie WDYT? should this be cjs?